Modules/CodeTrek/Http/Controllers/CodeTrekController.php

```diff
@@ -26,29 +26,27 @@ public function index(Request $request)
 
         return view('codetrek::index', ['centres' => $centres], $this->service->getCodeTrekApplicants($request->all()));
     }
-    /**
-     * Show the form for creating a new resource.
-     */
-    public function create()
-    {
-    }
 
     /**
      * Store a newly created resource in storage.
      */
-    public function store(Request $request, CodeTrekService $service)
+    public function store(Request $request)
     {
         $data = $request->all();
-        $applicant = $service->store($data);
+        $this->service->store($data);
 
         return redirect()->route('codetrek.index');
     }
 
     /**
      * Show the specified resource.
      */
-    public function show($id)
+    public function storeFeedback(Request $request)
     {
+        $data = $request->all();
+        $this->service->storeFeedback($data);
```

Comment:
I think we can directly write like this.
```suggestion
        $this->service->storeFeedback($request->all());
```